SpatialGDK/Source/SpatialGDK/Private/Utils/SpatialLatencyTracer.cpp

```diff
@@ -247,16 +286,28 @@ bool USpatialLatencyTracer::BeginLatencyTrace_Internal(const AActor* Actor, cons
 
 	WriteKeyFrameToTrace(&NewTrace, FString::Printf(TEXT("Begin trace : %s"), *FunctionName));
 
+	// For non-spatial tracing
+	const improbable::trace::SpanContext& TraceContext = NewTrace.context();
+
+	TArray<uint8> TraceBytes = TArray<uint8_t>((const uint8_t*)&TraceContext.trace_id()[0], sizeof(improbable::trace::TraceId));
+	TArray<uint8> SpanBytes = TArray<uint8_t>((const uint8_t*)&TraceContext.span_id()[0], sizeof(improbable::trace::SpanId));
+	OutLatencyPayload = FSpatialLatencyPayload(MoveTemp(TraceBytes), MoveTemp(SpanBytes));
+
 	TraceMap.Add(Key, MoveTemp(NewTrace));
 
+	if (!GetDefault<UGeneralProjectSettings>()->UsesSpatialNetworking())
+	{ // We can't do any deeper tracing in the stack here so terminate these traces here
```

Comment:
Total nit, but can you move the comment to a new line, here and elsewhere in this file.